docs/concepts/projects.md

```diff
@@ -33,24 +33,24 @@ The `deployment.yaml` file contains default configuration for all deployments cr
 The base structure for `deployment.yaml` is as follows:
 
 ```yaml
-# base metadata
-name: null
-version: null
-tags: []
-description: null
-schedule: null
-
-# flow-specific fields
-flow_name: null
-entrypoint: null
-parameters: {}
-parameter_openapi_schema: null
-
-# infra-specific fields
-work_pool:
-  name: null
-  work_queue_name: null
-  job_variables: {}
+deployments:
+  - # base metadata
+    name: null
+    version: null
+    tags: []
+    description: null
+    schedule: null
+
+    # flow-specific fields
+    flow_name: null
+    entrypoint: null
+    parameters: {}
+
+    # infra-specific fields
+    work_pool:
+    name: null
+    work_queue_name: null
+    job_variables: {}
```

Comment:
`name`, `work_queue_name`, and `job_variables` should be indented, I think.

Interestingly, my `deployment.yaml` populates my fields in alphabetical order: 🤔 

```yaml
deployments:
- description: null
  entrypoint: null
  flow_name: null
  name: null
  parameters: {}
  schedule: null
  tags: []
  version: null
  work_pool:
    job_variables: {}
    name: null
    work_queue_name: null
```